### DISCUSSION TITLE
Is Zitadel a complete platform like PingID or Okta?

### DISCUSSION THREAD

**Discussion by vgtom** [zitadel/zitadel#1867] [General]
Hi All,

Is Zitadel an API first solution or a complete solution like PingID or Okta?
Can you tell me a commercial platform solution it most closely is equivalent to?

Thanks in advance
vtom

**Comment by fforootd** [comment]
Hi @vgtom 

We think of ZITADEL as platform IAM. 
ZITADEL provides great APIs for easy integration but also brings a lot of features like delegation, role based access control, login policies, hosted login site with customisation options…

ZITADEL is best compared to Auth0 or Keycloak because we are kind of the „sweetspot“ between those two. You get a great platform optimised for the cloud-native era which is easy to use, run and integrate. And you can run it in your own datacenter with all the benefits.

What ZITADEL is not is a workflow engine. We think your are better of if you utilise existing business process tools and integrate them to ZITADEL via the APIs. 

More info to what we envision for ZITADEL can you find [in this blog](https://zitadel.ch/blog/our-vision-for-zitadel).

Does this answer your question? If you have specific question we are happy to help you 😁

**Reply by vgtom** [reply]
yes thank you foir your reply...i think i understand better now.

i would like to contribute code...how do i start doing that?

**Reply by fforootd** [reply]
Best way to get started is to head over to the [CONTRIBUTING.md](https://github.com/caos/zitadel/blob/main/CONTRIBUTING.md)

Although our process is still a little rough around the edges 😁we welcome any questions and will more and more publish our roadmap and ideas over the next few weeks.

If you are keen to implement a feature or fix a bug head over to the issues, or engage with @fgerschwiler or @livio-a here

**Reply by vgtom** [reply]
Hello @fgerschwiler  and @livio-a ...is there a way for to open a communication channel? i would like to work with you in contrbbuting code to zitadel!!

**Reply by fforootd** [reply]
Normally we rely on GH Issues and Discussion, and do not (yet) have a chat channel. 
Although it might be a great idea to start a such chat. Would you have any preference? Otherwise we will evaluate something quickly 😁

**Reply by vgtom** [reply]
I think as an open source project a slack channel is an absolute must...So i definitely prefer slack

**Reply by fforootd** [reply]
True, OSS projects often use slack with some kind of self-invite.
Let me check this

**Reply by fforootd** [reply]
Hi again we quickly did a discussion on how to achieve this and the TL;DR is that we want to prevent going to slack. This because it makes the community t0o locked in and not openly searchable. However we think we could rely on GitHub discussion for context based discussion. Natural we would make sure that we shift our internal discussion here as well.

How does this sound?

**Reply by fforootd** [reply]
This relates to this issue https://github.com/caos/zitadel/issues/1892
